docs/quickstart.md

```diff
@@ -36,7 +36,7 @@ docsify serve docs
 
 ## Manual initialization
 
-If you don't like `npm` or have trouble installing the tool, you can manually create `index.html`:
+If you don't like `npm` or have trouble installing the tool, you can manually create `index.html` (the code below will load the latest version of docsify automatically. If you want a specific version to avoid any broken elements in the future, [see below](#specifying-docsify-versions)):
 
 ```html
 <!-- index.html -->
```

Comment:
Both the `<link>` and `<script src="...">` elements should be locked to v4 in this HTML block.

```html
<link rel="stylesheet" href="//cdn.jsdelivr.net/npm/docsify@4/themes/vue.css">
```

```html
<script src="//cdn.jsdelivr.net/npm/docsify@4"></script>
```

Reply:
Oh, I assumed we still wanted the latest version block. No worries, will edit that asap!